xdem/biascorr.py

```diff
@@ -0,0 +1,113 @@
+"""Bias corrections for DEMs"""
+from __future__ import annotations
+
+from typing import Callable, Optional
+
+import numpy as np
+import rasterio as rio
+
+import xdem
+
+
+class DirectionalBias(xdem.coreg.Coreg):
+    """
+    For example for DEM along- or across-track bias correction.
+    """
+
+    def __init__(self, bias_func : Callable = xdem.robust_stats.robust_polynomial_fit):  # pylint: disable=super-init-not-called
+        """
+        Instantiate an directional bias correction object.
+
+        :param bias_func: The function to fit the bias. Default: robust polynomial of degree 1 to 6.
+        """
+        super().__init__(meta={"bias_func": bias_func})
```

Comment:
After this, `self._is_affine` should be set to False to work optimally with all parts of `Coreg`!

Reply:
Added